lib/rouge/lexer.rb

```diff
@@ -504,12 +526,22 @@ def self.detect?(text)
   end
 
   module Lexers
-    @_loaded_lexers = {}
+    def self.preload_lexer(tag)
+      lang = Lexer.find(tag)
+      if lang
+        lang.lexer_class
+      else
+        # needed during generate:cache
+        load "#{__dir__}/lexers/#{tag}.rb"
```

Comment:
```suggestion
        load File.join(__dir__, 'lexers', "#{tag}.rb")
```

Reply:
`File.join` actually uses `/` on Windows as well - there's not much point in it unless the elements may end or begin in slashes.